### ISSUE TITLE
I still have a question about the loss function you define on training this model

### ISSUE THREAD

**Issue by yansd-c** [xiph/rnnoise#80]
hi, sir. 
I have downloaded your code and got the whole training and evaluation process work.
But reading the code, I find that in the code you generate the data to train the network, you set the label--g[i], to -1 when there is noise only or there is no voice.
And when you define the loss funcion--mycost, you use K.sqrt(y_true) in it.
How can tensorflow or keras do sqrt(-1)? It seems impossible to me.
But it confuses me that I can train the model and get good denoising effect by using your code.
How does that work?????
